chart/values.yaml

```diff
@@ -391,6 +392,9 @@ scheduler:
   # Annotations to add to scheduler kubernetes service account.
   serviceAccountAnnotations: {}
 
+  # Launch additional containers into scheduler.
+  extraContainers: []
+
   # Mount additional volumes into scheduler.
   extraVolumes: []
```

Comment:
How to mount scheduler dags folder to s3-sync? Any example?

Reply:
@gardnerdev You can follow `Using additional containers` document in `README.md`.
An example S3 sync container is as follows: https://github.com/Swalloow/s3-sync
If you use EKS, IAM roles for service account settings are required.

```
extraContainers:
  - name: s3-sync
    image: myrepository/s3sync:latest
    imagePullPolicy: Always
    volumeMounts:
      - name: dags
        mountPath: /opt/airflow/dags
    env:
      - name: AWS_BUCKET
        value: airflow-src
      - name: KEY_PATH
        value: dags
      - name: DEST_PATH
        value: /opt/airflow/dags
      - name: INTERVAL
        value: "10"
```

Reply:
Thanks, I am using [ https://github.com/houqp/objinsync ]( https://github.com/houqp/objinsync ) but thanks to you I was able to configure it!